src/sentry/api/serializers/models/integration.py

```diff
@@ -76,16 +91,18 @@ def serialize(self, obj, attrs, user, include_config=True):
 
 
 @register(OrganizationIntegration)
-class OrganizationIntegrationSerializer(Serializer):
-    def __init__(self, params=None):
+class OrganizationIntegrationSerializer(Serializer):  # type: ignore
```

Comment:
why `type: ignore`?

Reply:
Every time I try to subclass `Serializer` I get this error:
```
src/sentry/api/serializers/models/integration.py:94: error: Class cannot subclass 'Serializer' (has type 'Any')
```

Reply:
You can ignore this for now. Once the `Serializer` class is typed this ignore can be removed.